rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_core_ca.h

```diff
@@ -112,6 +112,18 @@ static __asm    uint32_t __get_PSP (void) {
   sub   r0, r0, #32
   bx    lr
 }
+#elif (defined (__ARMCC_VERSION) && (__ARMCC_VERSION >= 6010050)) || defined ( __GNUC__ )
```

Comment:
In the conventional writing style, "arm" and "thumb" are mixed when "-mthumb" option is specified, and it does not work properly.
I wrote the whole function including the return value with the inline assembler.